client/src/components/SortDirection/SortDirection.js

```diff
@@ -2,6 +2,7 @@ import React from "react";
 
 import { trivial_text_maker } from "src/models/text";
 
+// eslint-disable-next-line import/no-unresolved
 import "./SortDirection.scss";
```

Comment:
This eslint-disable shouldn't be necessary... Try removing it, if you still get the warning, and can't sort it out, let me know.

Reply:
Done!